snippets/fuzzySearch.md

```diff
@@ -0,0 +1,33 @@
+### fuzzySearch
+
+Determines if the `patrn` matches with `str`
+
+Loops through `str` and determines if it contains all characters of `patrn` and in the correct order. Both the strings are converted to lower case.
+
+Taken from [here](https://github.com/forrestthewoods/lib_fts/blob/80f3f8c52db53428247e741b9efe2cde9667050c/code/fts_fuzzy_match.js#L18).
+``` js
+fuzzySearch = (patrn, str) => {
+    const pattern = patrn;
+    const string = str; 
+    let patternIdx = 0;
+    let strIdx = 0;
+    let patternLength = pattern.length;
+    let strLength = string.length;
+
+    while (patternIdx !== patternLength && strIdx !== strLength) {
```

Comment:
There are other looping choices that might be a better fit

PS: This specifies that it MUST be in order of pattern. There is also a way to do this same search as unordered, but I'm not sure that adhere's to what a fuzzy search is.

Reply:
I kept it short to adhere to the contribution guidelines. This is not a complete fuzzy search implementation.